source/fundamentals/serialization/class-mapping.txt

```diff
@@ -95,9 +98,36 @@ Customize Class Serialization
 -----------------------------
 
 You can customize how the driver serializes and deserializes documents at the class
-level by using attributes with the class or by calling methods while registering
+level by applying attributes to the class or by calling methods while registering
 a class map.
 
+.. _csharp-class-map-omit-fields:
+
+Omit Fields
+~~~~~~~~~~~
+
+By default, the driver serializes all fields in your POCOS, whether you
+define them or not. You can prevent specified fields from being
```

Comment:
By "define" you probably mean map/automap?. As a field in POCO is it's definition, therefore always defined.
I'd just omit that part.